CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 **IMPORTANT**: scikit-bio is no longer compatible with Python 2. scikit-bio is compatible with Python 3.4 and later.
 
 ### Features
+* Added trailing whitespace when reading runon GF/GS lines with `skbio.io.format.stockholm` reader. ([#1328](https://github.com/biocore/scikit-bio/issues/1328))
```

Comment:
Also, can you update the format docs to describe this new behavior? There's already note sections for how multi-line GF/GS features are handled so just update that text, and note that trees are not handled this way in the GF note.